tests/foreman/api/test_subscription.py

```diff
@@ -240,6 +243,60 @@ def test_positive_subscription_status_disabled(self):
         host_content = entities.Host(id=host_id).read_raw().content
         assert 'Simple Content Access' in str(host_content)
 
+    @pytest.mark.tier2
+    @pytest.mark.usefixtures("golden_ticket_host_setup")
+    @pytest.mark.usefixtures("rhel77_contenthost_class")
```

Comment:
Please switch these to single-quote strings. There are other instanced in this PR